src/pgrn-custom-scan.c

```diff
@@ -352,14 +360,45 @@ PGrnCollectScanKeySources(Relation index, List *quals)
 	return scanKeySources;
 }
 
+static bool
+PGrnIndexContainColumn(Relation index, const char *name)
+{
+	TupleDesc tupdesc = RelationGetDescr(index);
+	for (AttrNumber i = 0; i < tupdesc->natts; i++)
+	{
+		Form_pg_attribute attr = TupleDescAttr(tupdesc, i);
+		if (strcmp(NameStr(attr->attname), name) == 0)
+			return true;
+	}
+	return false;
+}
```

Comment:
PGrnIndexContainColumn:
I just changed the location of the existing function definition.